google/resource_google_project_service.go

```diff
@@ -0,0 +1,87 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceGoogleProjectService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGoogleProjectServiceCreate,
+		Read:   resourceGoogleProjectServiceRead,
+		Delete: resourceGoogleProjectServiceDelete,
+
+		Schema: map[string]*schema.Schema{
+			"service": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"project": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceGoogleProjectServiceCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	srv := d.Get("service").(string)
+
+	if err = enableService(srv, project, config); err != nil {
+		return fmt.Errorf("Error enabling service: %s", err)
+	}
+
+	d.SetId(srv)
```

Comment:
Should we include the project in this, so multiple projects can enable the same service within the same Terraform state?

Reply:
Sure! Done.